GVFS/GVFS.UnitTests/Git/GitProcessTests.cs

```diff
@@ -14,5 +14,80 @@ public void TryKillRunningProcess_NeverRan()
             GitProcess process = new GitProcess(new MockGVFSEnlistment());
             process.TryKillRunningProcess().ShouldBeTrue();
         }
+
+        [TestCase]
+        public void ResultHasNonWarningErrors_SingleLine_AllWarnings()
+        {
+            GitProcess.Result result = new GitProcess.Result(
+                string.Empty,
+                "warning: this line should not be considered an error",
+                1);
+
+            result.HasErrors.ShouldBeTrue();
+            result.HasNonWarningErrors.ShouldBeFalse();
+        }
+
+        [TestCase]
+        public void ResultHasNonWarningErrors_Multiline_AllWarnings()
+        {
+            GitProcess.Result result = new GitProcess.Result(
+                string.Empty,
+                @"warning: this line should not be considered an error
+warning: neither should this.",

```

Comment:
Do we need tests with spaces before "warning:"  or casing differences? I saw a couple of messages in the git source that use "Warning:" and "WARNING:"